src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1272,4 +1272,13 @@
   <data name="PreferTypedStringBuilderAppendOverloadsRemoveToString" xml:space="preserve">
     <value>Remove the ToString call</value>
   </data>
+  <data name="PreferConstCharOverConstUnitStringInStringBuilderDescription" xml:space="preserve">
+    <value>Prefer const char over const unit length string in StringBuilder.Append for more performant code</value>
+  </data>
+  <data name="PreferConstCharOverConstUnitStringInStringBuilderMessage" xml:space="preserve">
+    <value>Prefer const char over const unit length string</value>
```

Comment:
We normally use a more specific message with specifics about violations so the message is useful even from command line builds. See other examples above.